docker/Dockerfile

```diff
@@ -35,8 +35,10 @@ RUN  update-alternatives --install /usr/bin/clang clang /usr/bin/clang-$CLANG_VE
 
 RUN ln -s /usr/bin/clangd-${CLANG_VERSION} /usr/bin/clangd
 
-RUN echo "deb [trusted=yes] https://download.eclipse.org/zenoh/debian-repo/ /" | tee -a /etc/apt/sources.list.d/zenoh.list > /dev/null && \
-    apt-get update && apt-get install -y systemctl zenoh && \
+RUN export ZENOH_VERSION="0.11.0-rc.3" && \
```

Comment:
to make this change available in CI you need to bump the version in `version.sh`